cleverpush/src/main/java/com/cleverpush/CleverPush.java

```diff
@@ -148,6 +148,19 @@ public static CleverPush getInstance(@NonNull Context context) {
     private boolean showingTopicsDialog = false;
     private boolean confirmAlertShown = false;
 
+    private int currentPositionOfTagToAdd = 0;
+    private int currentPositionOfTagToRemove = 0;
+    private String[] tagIdsToAdd;
```

Comment:
might as well initialize both class members with default values, like with all values

(tagIdsToAdd)

(tagIdsToRemove)

Reply:
@YasinY @HardikLakum  This should not be a Class variable, right? This can't be a method-scope variable?